VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs

```diff
@@ -499,5 +502,24 @@ private string GetSettingsFilePath()
             var prefix = _httpContextAccessor.HttpContext.Request.Query["preview_mode"];
             return prefix.ToString().IsNullOrEmpty() ? "settings_data.json" : $"drafts\\{prefix}_settings_data.json";
         }
+
+        public bool IsFeatureActive(string featureName)
+        {
+            return _memoryCache.GetOrCreateExclusive(featureName, cacheEntry =>
```

Comment:
Better create cache key by this statement. This is guarantee its uniqueness
```
 var cacheKey = CacheKey.With(GetType(), nameof(IsFeatureActive), featureName);
```

Reply:
done